### DISCUSSION TITLE
Why is the vram of the graphics card not being fully utilized?

### DISCUSSION THREAD

**Discussion by TingQQ** [lllyasviel/Fooocus#1109] [Q&A]
My GPU is 3090 and its vram is 24GB.
I found out when I am running the fooocus image task, the GPU vram only use about 6GB.
![231653](https://github.com/lllyasviel/Fooocus/assets/90813057/e58ee067-12d1-4b24-b118-b3e6646a50c2)

From log shows:
Total VRAM 24575 MB, total RAM 16312 MB
Set vram state to: NORMAL_VRAM

How can I configure it to use the entire memory for execution?

**Comment by linzez** [comment]
I also want to know
